data/event-schemas/examples/m.room.topic.yaml

```diff
@@ -3,6 +3,14 @@
   "type": "m.room.topic",
   "state_key": "",
   "content": {
-    "topic": "A room topic"
+    "m.topic": {
+      "m.text": [ {
+        "mimetype": "text/html",
+        "body": "An <em>html</em> topic"
+      }, {
+        "body": "A plain text topic"
```

Comment:
I am not sure if it is wise to have different topic texts depending on the mime type as an example.

Reply:
Good point, yes. You shouldn't practically use it like this. Have updated the example to use markup around the same text instead.